rs/BUILD.bazel

```diff
@@ -1,6 +1,22 @@
 # The goal of using package_groups is to decouple semantically different parts of the whole IC codebase.
 # There is a 1-to-1 mapping between each package and a package group defined in this file.
 # The relationship between package_groups is also defined here.
+#
+#               vm-system-tests  ----
+#               |    \               |
+#               |    vm-ic-os        |
+#               |          \         |
+#               |         release    |
+#               |          |     \   |
+#               |          |     tools ( + integrations)
+#               |          |         |
+#               |          |         |
+#               |          |         |
+#               ======ic-protocol=====
+#               |          |         |
+#               BN     replica   dapps
```

Comment:
What do the `vm` in those package mean? Maybe consider using the same name as the package_groups.

Reply:
yes, I will rename the packages in follow-up MRs. 

I guess I wanted to emphasize which packages deal with VMs and which should not.